css/pages/cart/cart.css

```diff
@@ -0,0 +1,100 @@
+@import url('/css/base/base.css');
+@import url('/css/base/typography.css');
+@import url('/css/base/utilities.css');
+@import url('/css/layout/main-nav.css');
+@import url('/css/pages/promos.css');
+@import url('/css/pages/featured.css');
+@import url('/css/pages/footer.css');
+@import url('/css/layout/product-nav.css');
+@import url('/css/layout/product-footer.css');
+
+.cart-management {
+    height: 100vh;
+    padding: 3rem 0;
+}
+
+.cart-main-container {
+    gap: 2rem;
+    padding: 2rem;
+}
+
+.product-system {
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+    padding: 2rem;
+    width: 60%;
+    margin: 0 2rem;
+    height: 50rem;
+    overflow: auto;
+}
+
+.price-system {
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+    padding: 2rem;
+    width: 40%;
+    margin: 0 2rem;
+    height: 30rem;
+}
+
+/***********************/
+
+.basic-card {
+    width: 100%;
+    box-shadow: none;
+    height: 23rem;
+}
+
+.card-img-header {
+    height: 23rem;
+    width: 30%;
+}
+
+.header-text {
+    padding: 2rem;
+
+}
+
+/**********************/
+.cart-qnt {
+    padding: 3rem 0;
+}
+
+.plus,
+.minus,
+.qnt {
+    padding: 1rem;
```

Comment:
You should use utility classes for spacing as created in component library to be more consistent.